src/configure/configure.ts

```diff
@@ -121,34 +122,45 @@ class Orchestrator {
             await this.getAzureSession();
             await this.getSelectedPipeline();
 
-            if (this.inputs.pipelineConfiguration.template.label === "Containerized application to AKS") {
-                // try to see if node corresponds to any parameter of selected pipeline.
-                if (resourceNode) {
-                    let resourceParam = TemplateParameterHelper.getMatchingAzureResourceTemplateParameter(resourceNode, this.inputs.pipelineConfiguration.template.parameters);
-                    if (resourceParam) {
-                        this.inputs.pipelineConfiguration.params[resourceParam.name] = resourceNode;
+            
+            if(this.inputs.pipelineConfiguration.templateInfo) {
+                let extendedPipelineTemplate = await templateHelper.getTemplateParameteres(this.inputs.pipelineConfiguration.templateInfo);  
+                let inputs:{ [key: string]: any} = {};
+                inputs['subscriptionId'] = this.inputs.subscriptionId;
+                let controlProvider = new InputControlProvider(extendedPipelineTemplate, inputs);
```

Comment:
`inputs` I think is being used as a global context if I am not wrong(I might be wrong, please dispute if so). It is now clashing with inputs defined in template. Should we think about renaming it to `context`?

Reply:
Ohh you have defined a local variables named `inputs`. This is really confusing. Please handle.

Reply:
renamed local variable to context.